scripts/deploy.sh

```diff
@@ -126,9 +132,12 @@ helm_values_map["bpa.config.titleOverride"]=\"$BPA_TITLE_OVERRIDE\"
 helm_values_map["bpa.config.i18n.locale"]=$BPA_I18N_LOCALE
 helm_values_map["ux.preset"]=$UX_PRESET
 helm_values_map["ux.config.theme.themes.light.primary"]=$UX_PRIMARY_COLOR
+helm_values_map["ux.config.theme.themes.light.icons"]=$UX_ICONS_COLOR
 
 ## image
-helm_values_map["bpa.image.tag"]=$BPA_IMAGE_TAG
+declare BPA_VAR_IMAGE_TAG=${ENVIRONMENT^^}_IMAGE_TAG
+declare BPA_IMAGE_TAG_OVERRIDE=${IMAGE_TAG_OVERRIDE:-${!BPA_VAR_IMAGE_TAG}}
+helm_values_map["bpa.image.tag"]=${BPA_IMAGE_TAG_OVERRIDE}
```

Comment:
because the env file is loaded first, and then the config file,  if you use the same variable name in both, it will just use the one in the config. It's a bit of magic, but it means the script doesn't have to care about the source. 

Unless your intention was to have `DEV_IMAGE_TAG`, `TEST_IMAGE_TAG`, and `PROD_IMAGE_TAG` in each config, which seems unnessary given all deployments in a given OCP4 namespace should share the same tag (`sandbox`) being an exception.